exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceRuntime.java

```diff
@@ -208,37 +209,70 @@ public void initiateAddingService(Fork fork, ServiceInstanceSpec instanceSpec,
   }
 
   /**
-   * TODO(ECR-3919): fix the documentation of the method
-   * Adds a service instance to the runtime after it has been successfully initialized
-   * in {@link #initiateAddingService(Fork, ServiceInstanceSpec, byte[])}. This operation
-   * completes the service instance registration, allowing subsequent operations on it:
+   * Modifies the state of the given service instance at the runtime either by activation it or
+   * stopping. The service instance should be successfully initialized
+   * by {@link #initiateAddingService(Fork, ServiceInstanceSpec, byte[])} in advance.
+   * Activation leads to the service instance registration, allowing subsequent operations on it:
    * transactions, API requests.
+   * Stopping leads to the service disabling i.e. stopped service does not execute transactions,
+   * process events, provide APIs, etc. And it becomes unavailable to other services,
```

Comment:
The latest sentence is rather unclear. What becomes unavailable? I thought database indexes are still available after service stopping. Service interface methods (i.e. transactions) becomes unavailable, but we do not have support for them in EJB.